### PR TITLE
README.md: replace CircleCI badge with TC

### PR DESCRIPTION
**Pull request by dt**
Closes #10513.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/11207)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ![logo](docs/media/cockroach_db.png?raw=true "Cockroach Labs logo")
```
Comment (tamird) [inline README.md]: this link is broken, i just noticed =/

**Review by tamird — APPROVED** [review]

**Review by jordanlewis — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (dt) [inline README.md]: elaborate? it seems to click though to the logo for me.

Comment (tamird) [inline README.md]: weird, it's only broken in the PR view. carry on.

**Review by tamird — APPROVED** [review]

**Pull request merged**